File: 2026-08/src/day_25/prms_120839/Solution.java
package day_25.prms_120839;

class Solution {
    public String solution(String rsp) {
        StringBuilder sb = new StringBuilder();
        for (char c : rsp.toCharArray()) {
            if (c == '0') {
                sb.append("5");
            } else if (c == '2') {
                sb.append("0");
            } else {
                sb.append("2");
            }
        }

        return sb.toString();
    }
}

File: 2026-08/src/day_25/prms_120839/Solution.cpp
#include <bits/stdc++.h>
using namespace std;

string solution(string rsp) {
    string s;
    for (char c : rsp) {
        if (c == '0') {
            s += '5';
        } else if (c == '2') {
            s += '0';
        } else {
            s += '2';
        }
    }

    return s;
}
